third_party/move/move-compiler-v2/tests/checking/typing/recursive_structs_malformed.exp

```diff
@@ -16,7 +16,7 @@ error: reference type `&Bar` is not allowed as a field type
   │
   = required by declaration of field `f`
 
-error: tuple type `(&Baz, &mut Baz)` is not allowed as a type argument
+error: tuple type `(&Baz, &mut Baz)` is not allowed as a type instantiation
```

Comment:
The docs refer to this as a "type argument" or "type parameter".  See https://aptos.dev/move/book/generics/.